### ISSUE TITLE
Applying comps is confusing to new users 

### ISSUE THREAD

**Issue by Quinn-D** [neinteractiveliterature/intercode#2822]
The process of a GM applying comps is unclear and not logical for new to intercode 2 GMs in two areas.

-It is not obvious that they need to edit GMs in order to apply a comp to themselves
-The method for GMs to apply a comp to a non GM is rather a hack/workaround which is not easy for general users (add the person as a GM, apply comp, remove as GM)

Propose that a third link be added to the GM view of a game page, after "Edit Game", "Edit GMs".  For instance "Event Comp(s)".  Have this lead to functionally that allows GM to see how many are unassigned, who any are assigned to, and a function to apply to any user even if they are not a GM

**Comment by jhartwin** [comment]
Hmm. I'm not sure it makes sense to make it be this easy to apply comps to anyone. Of course we'll continue to apply them manually if asked, and the workaround you describe will continue to work. But in my mind, the "intended" use of GM comps is for the GMs themselves. While the con is ok with them being used for other people, and certainly wants to enable people to come to con who might not otherwise be able to afford it, I don't know that we want to openly encourage GMs to spend their comps on anyone and everyone.

To be clear, I'm not categorically opposed to the idea, I just want to think about it before implementing something like this. And we absolutely should improve documentation on how to use comps; that won't require a code change and the work is already in progress, but I wanted to state here that my caution here should not imply that I'm somehow against that plan.

**Comment by nbudin** [comment]
Regardless of Intercon's policies on this, I think there's a usability issue here that, as a platform owner, we should be solving.  The easiest fix I can think of would be to change the text from "Edit GMs" to something like "Edit GMs/Provide Badges".  That's not the prettiest possible fix, but would certainly make the feature more discoverable for folks.

@marleighnorton, any thoughts about this?

**Comment by valleyviolet** [comment]
I just want to note that I was caught by this usability issue myself because I wasn't careful how I apportioned the comps for my games (I don't want to give any extras to non-GMs but I need to give a badge to someone who is not GMing the game I have a badge left on). I would not have figured out the workflow that Quinn mentions by myself.

**Comment by Quinn-D** [comment]
@jhartwin I note that the GM policies does not make any direct encouragement that they only be used for GMs.  After details about numbers of comps, the text reads "These comped memberships can be assigned by the GM team in any manner they see fit."

**Comment by jhartwin** [comment]
Absolutely, and I'm not suggesting a policy change at all. I just don't know if I want the website to automatically guide GMs through applying comps to other non-GM attendees. I'm fine if they continue to use the workaround, and if they ask for them I'm fine for con staff to apply them manually. I just think that slight extra bit of friction might actually be better for the con, given that we are a non-profit funded entirely by attendance fees.

As Nat says, this is not actually the venue for discussion of Intercon policies. I'm fine with the fix Nat proposed for the usability issue. I'm also happy to continue this converstion if you'd like, @Quinn-D , but if so, let's move it to Slack.

**Comment by marleighnorton** [comment]
@nbudin Happy to weigh in, but I'm not a GM.  How do I view the problem?
